account_tax_balance/wizard/open_tax_balances.py

```diff
@@ -31,7 +31,23 @@ def onchange_date_range_id(self):
     def open_taxes(self):
         self.ensure_one()
         action = self.env.ref('account_tax_balance.action_tax_balances_tree')
-        vals = action.read()[0]
+        act_vals = action.read()[0]
+        # override action name doesn't work in v12 or v10
+        # we need to build a dynamic action on main keys
+        vals = {x: act_vals[x] for x in act_vals
+                if x in ('res_model', 'view_mode', 'domain',
+                         'view_id', 'search_view_id', 'name', 'type')}
+        lang = self.env['res.lang'].search(
+            [('code', '=', self.env.user.lang or 'en_US')])
+        date_format = lang and lang.date_format or "%m/%d/%Y"
+        infos = {'name': vals['name'], 'target': _(self.target_move),
+                 'from': self.from_date.strftime(date_format),
+                 'to': self.to_date.strftime(date_format),
+                 'company': self.company_id.name}
+        # name of action which is displayed in breacrumb
+        vals["name"] = _(
+            "%(name)s: %(target)s from %(from)s to %(to)s"
+            " (%(company)s)") % infos
```

Comment:
IMO the company should be displayed only in case the user belongs to the "Multi Companies" group.